doc/classes/CanvasItem.xml

```diff
@@ -423,6 +423,7 @@
 			<return type="Vector2" />
 			<description>
 				Returns the mouse's position in the [CanvasLayer] that this [CanvasItem] is in using the coordinate system of the [CanvasLayer].
+				[b]Note:[/b] For screen-space coordinates (e.g. when using non-embedded [Popup]), you can use [method DisplayServer.mouse_get_position].
```

Comment:
Tiny nitpick, sorry:
```suggestion
				[b]Note:[/b] For screen-space coordinates (e.g. when using a non-embedded [Popup]), you can use [method DisplayServer.mouse_get_position].
```
Or `[Popup]s`, I guess.

Looks great otherwise!